projects/packages/my-jetpack/_inc/components/product-detail-card/style.module.scss

```diff
@@ -1,16 +1,24 @@
+.card {
+	// Card main wrapper
+	--product-card-bg-color: white;
+	--product-card-border-color: #d5d5d5;
+	--product-card-shadow: rgb(0 0 0 / 3%);
+	--product-card-border-radius: 4px;
+	
+	background-color: var(--product-card-bg-color);
+	border: 1px solid var(--product-card-border-color);
+	border-radius: var(--product-card-border-radius);
+	box-shadow: 0 2px 6px var(--product-card-shadow), 0 1px 2px var(--product-card-shadow);
+	max-width: 384px;
```

Comment:
Didn't get where this value com from. Can you point me, please?

Reply:
It was picked from the current implementation of the Jetpack plugin. Let me try to find it.

Reply:
I think it diffs from `Figma`, but I think it could be made later

Reply:
https://github.com/Automattic/jetpack/blob/006bb984df43a5fc02e5c95f6fb075e546aad803/projects/js-packages/components/components/pricing-card/style.scss#L6